cmd/crank/install.go

```diff
@@ -120,6 +129,30 @@ func (c *installConfigCmd) Run(k *kong.Context, logger logging.Logger) error {
 		logger.Debug("Failed to create configuration", "error", warnIfNotFound(err))
 		return errors.Wrap(warnIfNotFound(err), "cannot create configuration")
 	}
+	if err = waitUntil(waitDuration, func() error {
+		logger.Debug("Waiting for the configuration to be ready")
+		res, err := kube.Configurations().Get(context.Background(), pkgName, metav1.GetOptions{})
+		if err != nil {
+			logger.Debug("Failed to fetch created configuration", "error", err)
+			return errors.Wrap(err, "failed to fetch configuration")
+		}
+		var configStatus corev1.ConditionStatus
+		for _, condition := range res.Status.Conditions {
+			if condition.Type == v1.TypeHealthy {
+				configStatus = condition.Status
```

Comment:
after switching to conditional wait, you can just return here

Reply:
Looks like this one still needs to be addressed. You can also just use [`GetCondition()`](https://github.com/crossplane/crossplane/blob/d04a059fe341a941760a3a94ef07085ae7158365/apis/pkg/v1/interfaces.go#L90)